index.html

```diff
@@ -53,5 +53,257 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
         </ul>
       </section>
     </section>
+
+<section class="whole-middle-section">
+    <section class="recent-work">
+        <h1 class="recent-work-content">
```

Comment:
- [ ] Well done of using the section tag but what I suggest you do is, for example, the section of projects you have to put inside of a section than cards of projects you can wrap them with divs and not sections. Kindly change that 💪